services/harmony/app/markdown/apis.md

```diff
@@ -72,6 +79,60 @@ curl -Lnbj {{root}}/{{exampleCollection}}/ogc-api-coverages/1.0.0/collections/ba
 ```
 **Example {{exampleCounter}}** - Curl command for an OGC Coverages request
 
+#### OGC EDR Request Parameters
+
+The Harmony services REST API also conforms to the OGC EDR API version 1.1.0.
+As such it accepts parameters in the URL path as well as query parameters.
+Currently only the `/cube` route is supported for data search. Other EDR search routes will be supported in the near future.
+
+##### URL Path Parameters
+| parameter | description |
+|-----------|-------------|
+| collection | (required) This is the NASA EOSDIS collection or data product. There are two options for inputting a collection of interest:<br/>1. Provide a concept ID, which is an ID provided in the Common Metadata Repository (CMR) metadata<br/>2. Use the data product short name, e.g. SENTINEL-1_INTERFEROGRAMS. Must be URL encoded. |
+---
+**Table {{tableCounter}}** - Harmony OGC EDR API URL path (required) parameters
+
+<br/>
+<br/>
+
+##### Query Parameters
+| parameter | description |
```

Comment:
I'm thinking as we add more routes we should add a column in the middle to say which routes support it. So for `bbox` we would say `cube` and all the other parameters we would say `all`. For the `coords` parameter used in all the other routes we might need to explain in the description that each route requires a different WKT type, but otherwise I think it will work well and prevent a ton of duplication.